bugzoo/mgr/coverage.py

```diff
@@ -241,17 +242,17 @@ def extract(self,
             instrumented_files = set(instrumented_files)
 
         dir_source = container.bug.source_dir # TODO port
-        logger.debug("Running gcovr. Seconds passed: {}".format(timer() - t_start))
+        logger.debug("Running gcovr. Seconds passed: %.2f", timer() - t_start)
```

Comment:
Can we report the time taken to run `gcovr` rather than the total time taken by the instrumentation process? Other than that, it looks good.